features/gezag-minderjarige/4b.1-ouder-of-partner-overleden-of-onbevoegd-tot-gezag.feature

```diff
@@ -0,0 +1,457 @@
+#language: nl
+
+Functionaliteit: 4b.1 - Ouder of partner overleden of onbevoegd tot gezag
+  Precondities voor deze vraag:
+  | 1.1 Staat persoon (minderjarige) als ingezetene in de BRP?             | Ja                                                                       |
+  | 1.2 Is persoon a. minderjarig en b. niet overleden?                    | Ja                                                                       |
+  | 1.3 Is minderjarige naar het buitenland geëmigreerd geweest?           | Nee                                                                      |
+  | 1.3a Is minderjarige in buitenland geboren?                            | Nee of geadopteerd met NL akte                                           |
+  | 1.4 Is uitspraak gezag aanwezig                                        | Nee                                                                      |
+  | 2.1 Hoeveel juridische ouders heeft minderjarigen                      | Een_ouder                                                                |
+  | 2b.1 is staande huwelijk of partnerschap geboren                       | Ja                                                                      |
+
+  Onbevoegd tot het gezag zijn minderjarigen, zij die onder curatele zijn gesteld en zij wier geestvermogens zodanig zijn gestoord, om het
+  gezag uit te kunnen oefenen (1:246 BW). Daarnaast zijn overleden ouders ook onbevoegd tot het gezag.
+
+  Mogelijke uitkomsten:
+  1) Nee
+  -> GG: Ouder en echtgenoot/partner hebben gezamenlijk het gezag (1:253sa BW)
+  a. meemoeders: kind tijdens huwelijk/gp geboren, niet erkend en er is geen sprake van een verklaring onbekende donor
+  b. kind vóór 1 april 2014 geboren tijdens geregistreerd partnerschap van paren verschillend geslacht, kind is niet erkend (Zie 2b.1 onder 1)
+  2) Ja, de niet-ouder
+  -> OG1: De ouder heeft alleen het gezag (1:253f BW)
+  3) Ja, de ouder
+  -> V: De niet-ouder heeft alleen het gezag (1:253x BW).
+  4) Ja, beide ouders
+  -> G: Er is (tijdelijk) geen gezag (1:253g lid 1 BW
+
+  Op het moment dat de moeder weer bevoegd wordt tot het gezag, verkrijgt zij van rechtswege het gezag, tenzij op dat tijdstip een ander met het
```

Comment:
Hoe zit het met deze "tenzij op dat tijdstip een ander met het gezag is belast"?

Moet er dus worden gekeken of er eerder curatele geweest is op een ouder en wie gezag had op het moment dat curatele werd opgeheven?

Reply:
Geldt deze tekst voor zowel curatele als minderjarigheid?

Is er altijd wanneer "een ander met het gezag is belast" sprake van een uitspraak tot gezag (indicatie gezag minderjarige gevuld) of kan dit ook gezag van rechtswege voor een ander zijn?

Reply:
mee eens. Als ik [Artikel 253b](https://maxius.nl/burgerlijk-wetboek-boek-1/artikel253b) goed begrijp gaat die "tenzij..." over situaties dat er een gerechtelijke uitspraak tot gezag is. Daar zijn we in de gezagsbepaling al langs geweest op het moment dat vraag 4b.1 beantwoord moet worden. Dat stukje is hier m.i. dus niet relevant.